gtk/src/toga_gtk/widgets/activityindicator.py

```diff
@@ -23,10 +26,10 @@ def rehint(self):
             #     self.native.get_preferred_width(),
             #     self.native.get_preferred_height(),
             # )
-            width = self.native.get_preferred_width()
-            height = self.native.get_preferred_height()
-
-            self.interface.intrinsic.width = width[0]
-            self.interface.intrinsic.height = height[0]
+            width = self.native.get_preferred_width()[0]
+            height = self.native.get_preferred_height()[0]
         else:  # pragma: no-cover-if-gtk3
-            pass
+            size = self.native.get_preferred_size()[1]
```

Comment:
We should stay consistent and use the minimum size here.

```suggestion
            size = self.native.get_preferred_size()[0]
```